ui/analyse/src/wiki.ts

```diff
@@ -5,15 +5,33 @@ export type WikiTheory = (nodes: Tree.Node[]) => void;
 export default function wikiTheory(): WikiTheory {
   const cache = new Map<string, string>();
   const show = (html: string) => {
-    $('.analyse__wiki').html(html).toggleClass('empty', !html);
+    $('.analyse__wiki').toggleClass('empty', !html);
+    $('.analyse__wiki-text').html(html);
     site.pubsub.emit('chat.resize');
   };
 
+  $(window).on('load', () => {
+    const wikiBookField = $('#wikibook-field').first();
+    const wikiBookStorageKey = 'analyse.wikibooks.display';
```

Comment:
`const wikiBookStorage = site.storage.boolean('analyse.wikibooks.display')`

which you can then use like `wikiBookStorage.toggle()`

Reply:
Updated, thanks.